src/redux/modules/volunteer.ts

```diff
@@ -176,8 +197,17 @@ export const inviteVolunteer = (
   const action = await dispatch(
     setVolunteers([...state.all_volunteers, volunteer]),
   );
-  if (action.payload instanceof Array) {
+  if (action.type === 'volunteer/SET_VOLUNTEERS') {
```

Comment:
wanna just compare it to the  CONSTANT instead of hardcoding the string?

Reply:
Yes good call - just changed